library/src/androidMain/kotlin/dev/bluefalcon/BlueFalcon.kt

```diff
@@ -333,8 +333,10 @@ actual class BlueFalcon actual constructor(
                 return
             }
             gatt?.device?.let { bluetoothDevice ->
+                val bluetoothPeripheral = BluetoothPeripheral(bluetoothDevice)
+                bluetoothPeripheral.mtuSize = mtu
```

Comment:
do the Android bluetooth periphierial not have this property already?

Reply:
The actual library doesn't (unless I'm looking at the wrong documentation: https://developer.android.com/reference/android/bluetooth/BluetoothDevice). I treated it the same as the rssi where I created `actual var`